javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -41,6 +41,7 @@ const { Credential } = require('./virtual_authenticator')
 const webElement = require('./webelement')
 const { isObject } = require('./util')
 const BIDI = require('../bidi')
+const { PinnedScript } = require('./pinnedScript')
```

Comment:
I think this should be 
```suggestion
const { PinnedScript } = require('./pinnedscript')
```

The tests are failing because of this. could you have a look closer

Reply:
Thank you for bringing this to my notice. Locally, the file name was updated and I thought those changes were pushed. Somehow change in the file name was not detected. Was able to fix it on Github directly.